From 18f0a19160fa44e47e392d00e8b4f400c28bdf2a Mon Sep 17 00:00:00 2001
From: Wen He <wen.he_1@nxp.com>
Date: Thu, 31 Jan 2019 15:57:04 +0800
Subject: [PATCH 626/706] arm64: dts: ls1028ardb: Add resolution property

This patch add resolution property for LS1028ARDB board. It will
change the supported resolution list of the specify monitor.

Signed-off-by: Wen He <wen.he_1@nxp.com>
(cherry picked from commit af463c1db30391f54b5567228461c1bc7eeffe2c)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts b/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts
index bb672ab896d7..dc5460a0b15d 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts
@@ -299,6 +299,10 @@
 
 &edp {
 	fsl,no_edid;
+	resolution = "3840x2160@60",
+		   "1920x1080@60",
+		   "1280x720@60",
+		   "720x480@60";
 	lane_mapping = <0x4e>;
 	edp_link_rate = <0x6>;
 	edp_num_lanes = <0x4>;
-- 
2.17.1

